src/jit/target.h

```diff
@@ -2394,6 +2394,14 @@ C_ASSERT((RBM_INT_CALLEE_SAVED & RBM_FPBASE) == RBM_NONE);
 #endif
 /*****************************************************************************/
 
+#ifdef _TARGET_64BIT_
+typedef unsigned __int64 TargetPointerType;
```

Comment:
It seems weird to call this a "PointerType" when it is not a pointer type.

This is basically `size_t` for the target. Is there not something already like this? Or should it be named `target_size_t`?

Reply:
I did my best to find such type, but failed. And agree, my choice of name for this type is confusing.